src/objects/zcl_abapgit_objects.clas.abap

```diff
@@ -4,6 +4,14 @@ CLASS zcl_abapgit_objects DEFINITION
 
   PUBLIC SECTION.
 
+    TYPES:
+      BEGIN OF ty_supported_types,
+        obj_type  TYPE tadir-object,
+        supported TYPE abap_bool,
+      END OF ty_supported_types.
+
+    TYPES: ty_supported_types_tt TYPE SORTED TABLE OF ty_supported_types WITH UNIQUE KEY obj_type.
+
```

Comment:
I'd suggest moving these types to `PRIVATE SECTION`

Reply:
thanks. good hint. done